openedx/core/djangoapps/discussions/migrations/0008_auto_20211115_0821.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.8 on 2021-11-15 08:21
+
+from django.db import migrations, models
+import django_mysql.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('discussions', '0007_add_discussion_topic_link'),
+    ]
+
```

Comment:
can you please tell me why this new migration is added?

Reply:
rather than adding this migration, can you think of another way to exclude this provider? from the API?

Reply:
Sure, I can add a disabled, or hidden flag to the providers. Will do tht instead.